charts/bulk-scan-orchestrator/values.yaml

```diff
@@ -11,6 +11,7 @@ java:
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     TRANSFORMATION_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    TRANSFORMATION_URL_DIVORCE: "http://div-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
Should this be null by default, so that it's not called in production?

Reply:
Aaa, read your other email. Ok.

Reply:
yeah, in flux pr I set this value to null in production